CRISPResso2/CRISPRessoAggregateCORE.py

```diff
@@ -232,8 +232,8 @@ def main():
                     suffix_counter += 1
                 seen_names.append(amplicon_names[amplicon])
 
-            crispresso2_info['ref_names'] = seen_names
-            crispresso2_info['refs'] = {}
+            crispresso2_info['results']['ref_names'] = seen_names
+            crispresso2_info['results']['refs'] = {}
             crispresso2_info['summary_plot_names'] = []
```

Comment:
Let's move these down two levels to results/general_plots as well?

Reply:
You would like `results/refs` and `results/ref_names` to be `results/general_plots/refs` and `results/general_plots/ref_names`?

Reply:
Nah - I think they're fine where they are. Thanks so much.